### DISCUSSION TITLE
Keeping library and import names consistent

### DISCUSSION THREAD

**Discussion by ikwyl6** [PyMySQL/mysqlclient#709] [Q&A]
Hi - have you considered trying to keep the library name, import name and project name consistent?

You have ```mysqlclient``` as the github name, the src dir is called ```MySQLdb```, and the import name is ```MySQLdb```. 

I know there may be a few similar names out there for python mysql/mariadb libraries.. but would you open to changing this to something that is consistent across all areas?

I know when I was trying to find a python mariadb library that it was very hard to get it straight what lib I was looking for as some libs are outdated or not used and then there are also some libraries on pypi that might be totally separate from github as well. This might be an annoying topic for you as perhaps you already went through this in the past.. but I thought I would bring it up and see what you think.

**Comment by methane** [comment]
too late.

**Reply by ikwyl6** [reply]
How is it too late?

**Reply by methane** [reply]
Read #232.
For more histrical information, read #44.
